emoji_codepoints.py

```diff
@@ -1,4147 +1,8288 @@
 # -*- coding: UTF-8 -*-
```

Comment:
how is this file generated ?

can we instead check-in the generator and run it on the server at startup, to populate a data file mentioned in the config, if it doesn't exist or empty